composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":             ">=5.3.2",
         "ext-curl":        "*",
-        "guzzle/guzzle":   "~3.7"
+        "guzzlehttp/guzzle": "~3.7 || ~4.0 || ~5.0 || ~6.0"
```

Comment:
Maybe. It's up to maintainers decision.

BTW, library flexibility argument is still valid, even if we drop php <5.5 support.